debugging/book-library/script.js

```diff
@@ -52,50 +43,44 @@ function Book(title, author, pages, check) {
 
 function render() {
   let table = document.getElementById("display");
-  let rowsNumber = table.rows.length;
-  //delete old table
-  for (let n = rowsNumber - 1; n > 0; n-- {
-    table.deleteRow(n);
-  }
-  //insert updated row and cells
-  let length = myLibrary.length;
-  for (let i = 0; i < length; i++) {
-    let row = table.insertRow(1);
+  let tableBody = table.getElementsByTagName("tbody")[0];
+  tableBody.innerHTML = ''; // Clear all existing rows
+
+  for (let i = 0; i < myLibrary.length; i++) {
+    let row = tableBody.insertRow();
+
     let cell1 = row.insertCell(0);
     let cell2 = row.insertCell(1);
     let cell3 = row.insertCell(2);
     let cell4 = row.insertCell(3);
     let cell5 = row.insertCell(4);
+
     cell1.innerHTML = myLibrary[i].title;
     cell2.innerHTML = myLibrary[i].author;
     cell3.innerHTML = myLibrary[i].pages;
 
-    //add and wait for action for read/unread button
+   
     let changeBut = document.createElement("button");
-    changeBut.id = i;
     changeBut.className = "btn btn-success";
     cell4.appendChild(changeBut);
-    let readStatus = "";
-    if (myLibrary[i].check == false) {
-      readStatus = "Yes";
-    } else {
-      readStatus = "No";
-    }
+
+    let readStatus = myLibrary[i].check ? "No" : "Yes";
```

Comment:
I think the logic of assigning Yes/No should not be opposite to the value of this property.  In your case if `.check` property is 'truthy' you assign 'No' and visa versa. Please, consider updating this line.